tests/sklearn/test_sklearn_models.py

```diff
@@ -1139,6 +1163,50 @@ def check_load_fitted_sklearn_linear_models(model_class, n_bits, x, y, check_flo
     )
 
 
+def check_rounding_consistency(
+    model,
+    x,
+    y,
+    predict_method,
+    metric,
+    is_weekly_option,
+):
+    """Test that Concrete ML witout and with rounding are 'equivalent'."""
+
+    # Run the test with more samples during weekly CIs
+    if is_weekly_option:
+        fhe_samples = 5
+    else:
+        fhe_samples = 1
+
+    # Check that separated inference steps (encrypt, run, decrypt, post_processing, ...) are
```

Comment:
this comment is wrong I think, here the following line is only taking a random sample for the inferebce

Reply:
oh yes bad copy paste